### PR TITLE
Dx 924 gender neutral documentation

### PR DESCRIPTION
**Pull request by marinafrydenb**
Make the documentation gender neutral as requested in https://github.com/SwedbankPay/developer.swedbankpay.com/issues/372#issuecomment-623373794.

### CODE REVIEW

File: payments/payment-link.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ you include following information:
   sent  to the consumer).
 * Directions to (a link to a page) the merchant's terms and conditions (such as
   return policy) and information of how the consumer can contact the merchant.
-* Details informing the consumer that he or she accepts the Terms&Conditions
-  when clicking on the Payment Link.
+* Details informing the consumer that Terms&Conditions are concidered to be
```
Comment (NullableInt) [inline payments/payment-link.md:58]:
```suggestion
* Details informing the consumer that Terms & Conditions are concidered to be
```
Personal preference on this one.

File: _includes/payment-link.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ We recommend that you include following information:
   sent  to the consumer).
```
Comment (asbjornu) [inline _includes/payment-link.md:63]:
```suggestion
  sent  to the payer).
```

File: _includes/payment-link.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ We recommend that you include following information:
...
 * Directions to (a link to a page) the merchant's terms and conditions (such as
   return policy) and information of how the consumer can contact the merchant.
-* Details informing the consumer that he or she accepts the Terms & Conditions
+* Details informing that the consumer accepts the Terms & Conditions
```
Comment (asbjornu) [inline _includes/payment-link.md:66]:
```suggestion
* Details informing that the payer accepts the Terms & Conditions
```

File: payments/payment-link.md (hunk 1 above)
Comment (asbjornu) [inline payments/payment-link.md:58]:
```suggestion
* Details informing the consumer that Terms & Conditions are considered to be
```

File: payments/vipps/redirect.md [hunk 4]
```diff
@@ -204,9 +203,9 @@ of the payer towards the URL in the `href` of the `redirect-authorization`
 operation found in the list of `operations` in the response from the creation of
 the payment.
 
-Once the payment is successfully authorized by the payer, she is returned to
-`completeUrl` or `cancelUrl` depending on the action performed by the payer.
-On that page as well as in the `callbackUrl`, you need to perform an HTTP `GET`
+Once the payment is successfully authorized, the payer is returned to either the
+`completeUrl` or the `cancelUrl`; depending on the action performed.
+On the page as well as in the `callbackUrl` you need to perform a HTTP `GET`
```
Comment (asbjornu) [inline payments/vipps/redirect.md:208]:
```suggestion
On the page as well as in the `callbackUrl` you need to perform an HTTP `GET` 
```

**Pull request merged**